Scripts/script-ParseEmailFiles.yml

```diff
@@ -35,7 +35,6 @@ script: |-
   import olefile as OleFile
   from email.parser import HeaderParser
   from email.parser import Parser as EmailParser
-  import re
```

Comment:
this code is taken from somewhere open-sources so it should not be changes (all code under license)